code/extension/DataGridDataConvert/Source/NET/obj/Release/.NETFramework,Version=v4.6.1.AssemblyAttributes.cs

```diff
@@ -0,0 +1,4 @@
+// <autogenerated />
+using System;
+using System.Reflection;
+[assembly: global::System.Runtime.Versioning.TargetFrameworkAttribute(".NETFramework,Version=v4.6.1", FrameworkDisplayName = ".NET Framework 4.6.1")]
```

Comment:
@glundgren93 I was not the only one!
@gnbm I think we are not supposed t commit this file

Reply:
Yes, please remove this

Reply:
Done!

Reply:
@rugoncalves Do you know why this file was added? Just curious about it since it seems to be part of an OS extension